### PR TITLE
Add image blur

### PR DESCRIPTION
**Pull request by seekuehe**
Adds support for blurring images, including an effect to extend the blur beyond the original bounds of the image
 
There are two blur props applied, one to the image itself in `<a:blip>` and optionally under `<p:spPr>` for the bounding shape of the image.

**Example**
```js
let pptx = new PptxGenJS();
let slide = pptx.addSlide();

slide.addImage({
  x: 1,
  y: 2,
  path: "https://images.unsplash.com/photo-1506744038136-46273834b3fb?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMTIyMnwwfDF8c2VhcmNofDJ8fGxhbmRzY2FwZXxlbnwwfHx8fDE2NTg5MzE0MDM&ixlib=rb-1.2.1&q=80&w=1080",
    blur: {radius: 20}
});

slide.addImage({
  x: 3,
  y: 2,
  path: "https://images.unsplash.com/photo-1506744038136-46273834b3fb?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMTIyMnwwfDF8c2VhcmNofDJ8fGxhbmRzY2FwZXxlbnwwfHx8fDE2NTg5MzE0MDM&ixlib=rb-1.2.1&q=80&w=1080",
    blur: {radius: 20, grow: true}
});

pptx.writeFile({ fileName: 'PptxGenJS-Blur.pptx' });
```

### CODE REVIEW

File: dist/pptxgen.cjs.js [hunk 1]
```diff
@@ -5563,7 +5564,7 @@ function slideObjectToXml(slide) {
                 strSlideXml += '</p:sp>';
                 break;
             case SLIDE_OBJECT_TYPES.image:
-                var sizing = slideItemObj.options.sizing, rounding = slideItemObj.options.rounding, width = cx, height = cy, fillOverlayXmlString = genXmlFillOverlay(slideItemObj.options.fillOverlay); genXmlImgSolidFill(slideItemObj.options.solidFill);
+                var sizing = slideItemObj.options.sizing, rounding = slideItemObj.options.rounding, width = cx, height = cy, fillOverlayXmlString = genXmlFillOverlay(slideItemObj.options.fillOverlay), solidFillXmlString = genXmlImgSolidFill(slideItemObj.options.solidFill), hasEffects = slideItemObj.options.shadow || slideItemObj.options.blur, imgBlur = slideItemObj.options.blur;
```
Comment (pastafari) [inline dist/pptxgen.cjs.js:5567]: Why are all these on one long line? Shall we split them on separate lines?

File: dist/pptxgen.cjs.js [hunk 2]
```diff
@@ -5611,22 +5614,33 @@ function slideObjectToXml(slide) {
...
+                        strSlideXml += ' blurRad="' + slideItemObj.options.shadow.blur + '"';
+                        strSlideXml += ' dist="' + slideItemObj.options.shadow.offset + '" dir="' + slideItemObj.options.shadow.angle + '">';
+                        strSlideXml += '<a:srgbClr val="' + slideItemObj.options.shadow.color + '">';
+                        strSlideXml += '<a:alpha val="' + slideItemObj.options.shadow.opacity + '"/></a:srgbClr>';
+                        strSlideXml += '</a:' + slideItemObj.options.shadow.type + 'Shdw>';
+                    }
+                    // EFFECTS > BLUR REF: @see http://officeopenxml.com/drwPic-effects.php
+                    // Blur grow has to be applied to the bounding shape
+                    // But should minimize compounding the blur effect on the image
+                    if (imgBlur === null || imgBlur === void 0 ? void 0 : imgBlur.grow) {
```
Comment (pastafari) [inline dist/pptxgen.cjs.js:5640]: I dont understand the condition here, why is there a ternary operator? What are we trying to say?

File: src/gen-xml.ts [hunk 3]
```diff
@@ -629,26 +633,41 @@ function slideObjectToXml(slide: PresSlide | SlideLayout): string {
...
+						strSlideXml += '<a:alpha val="' + slideItemObj.options.shadow.opacity + '"/></a:srgbClr>'
+						strSlideXml += '</a:' + slideItemObj.options.shadow.type + 'Shdw>'
+					}
+
+					// EFFECTS > BLUR REF: @see http://officeopenxml.com/drwPic-effects.php
+					// Blur grow has to be applied to the bounding shape
+					// But should minimize compounding the blur effect on the image
+					if (imgBlur?.grow) {
+						imgBlur.radius = valToPts(imgBlur.radius / 10 || 8)
+						strSlideXml += `<a:blur rad="${imgBlur.radius}" grow="1"/>`
```
Comment (pastafari) [inline src/gen-xml.ts:665]: So if I understand correctly, we apply `blur` to both `<blip>` and `<effectLst>` correct?

Comment (seekuehe) [reply src/gen-xml.ts:665]: For the grow effect, yes

**Review by pastafari — APPROVED** [review]
Tested on sandbox locally. LGTM! 🎉

**Pull request merged**